extensions/monobank/src/accounts.tsx

```diff
@@ -0,0 +1,312 @@
+    } else {
+      setPinned([...pinned, item.id]);
+      await showToast(Toast.Style.Success, `Pinned ${getTitle(item)}`);
+    }
+  }
+
+  async function onRearrange(item: Account | Jar, direction: "up" | "down") {
+    const accountIndex = pinned.findIndex((pinnedAccount) => pinnedAccount === item.id);
+    const newPinned = [...pinned];
+
+    if (direction === "up") {
+      newPinned[accountIndex] = newPinned[accountIndex - 1];
+      newPinned[accountIndex - 1] = item.id;
+      await showToast(Toast.Style.Success, `Moved up ${getTitle(item)}`);
+    } else {
+      newPinned[accountIndex] = newPinned[accountIndex + 1];
+      newPinned[accountIndex + 1] = item.id;
+      await showToast(Toast.Style.Success, `Moved down ${getTitle(item)}`);
+    }
+
+    setPinned(newPinned);
+  }
+
+  function toggleDetails() {
+    setIsShowingDetail((isShowingDetail) => !isShowingDetail);
+  }
+
+  async function onCopyTotal() {
+    await Clipboard.copy(totalAmount.toFixed(2));
+    await showHUD("Copied to Clipboard");
+  }
+
+  function getValidRearrangeDirections(item: Account | Jar) {
+    return {
+      up: pinned.findIndex((pinnedAccount) => pinnedAccount === item.id) > 0,
+      down: pinned.findIndex((pinnedAccount) => pinnedAccount === item.id) < pinned.length - 1,
+    };
+  }
+
+  const cards = accounts.filter((account) => account.type !== AccountType.FOP);
+  const fops = accounts.filter((account) => account.type === AccountType.FOP);
+
+  const pinnedAccounts = pinned
+    .map((pinnedAccountId) => [...accounts, ...jars].find((account) => account.id === pinnedAccountId))
+    .filter((account) => {
+      if (!account) return false;
+
+      if (isAccount(account)) {
+        return satisfiesTexts(
+          searchText,
+          account.title,
+          account.currency.code,
+          account.type,
+          account.maskedPan.length ? account.maskedPan[0] : account.iban
+        );
+      }
+
+      return satisfiesTexts(searchText, account.currency.code, account.title);
+    }) as (Account | Jar)[];
+
+  const filteredCards = filterOutPinnedItems({ category, items: cards, pinned }).filter((card) =>
+    satisfiesTexts(searchText, card.title, card.currency.code, card.type, card.maskedPan[0])
+  );
+
+  const filteredFops = filterOutPinnedItems({ category, items: fops, pinned }).filter((fop) =>
+    satisfiesTexts(searchText, fop.title, fop.currency.code, fop.type, fop.iban)
+  );
+
+  const filteredJars = filterOutPinnedItems({ category, items: jars, pinned }).filter((jar) =>
+    satisfiesTexts(searchText, jar.title, jar.currency.code)
+  );
+
+  const totalAmount = calculateTotal([...cards, ...fops, ...jars], rates);
+
+  const isLoading = isClientInfoLoading || isRatesLoading || isPinnedLoadingFromLS;
+
+  return (
+    <List
+      isLoading={isLoading}
+      isShowingDetail={isShowingDetail}
+      navigationTitle={!isRatesError ? `Total: ${formatCurrency(totalAmount, "UAH")}` : undefined}
```

Comment:
Don't change the navigationTitle in the root command - it will be automatically set to the command name. Use navigationTitle only in nested screens to provide additional context.

I'm not sure how we can show it the best way, as a dynamic command subtitle or another item in the list instead.

Reply:
@pernielsentikaer what if I use `environment.commandName` to get the command name and output it together with this info like:

`"Show Accounts (Total: <amount>)"` ? I just believe it's a perfect place to output the total of all accounts.

Or it is totally forbidden to put something in here?